lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -369,7 +369,8 @@ def _cf_simple_api_call(self,api_call,method='GET',payload=None):
 
         if content:
             try:
-                result = json.loads(content)
+                resp_json = content.decode('utf-8')
```

Comment:
Rather than using separate variables, you can accomplish this in one line:

`result = json.loads(content.decode('utf-8'))`